src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+
+    display: block;
+  }
+
+  &__header {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+  }
+
+  &__code {
+    font-size: $small-font-size;
+    color: $secondary-text-color;
+    line-height: 14px;
+
+    margin-bottom: 16px;
+  }
+
+  &__evaluation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: $small-font-size;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    color: $secondary-text-color;
+    line-height: 18px;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__buy-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    text-decoration: none;
+    font-weight: 700;
```

Comment:
Lets create a variable, looks like it was already used more than one time and can be used many times for bold font